src/env.js

```diff
@@ -70,6 +75,16 @@ const envDefinitions = {
     desc:
       "force implementation for ALL currency bridges (affects scanning accounts)",
   },
+  COSMOS_GAS_AMPLIFIER: {
+    def: 2,
+    parser: intParser,
+    desc: "estimate gas multiplier",
+  },
+  COSMOS_GAS_PRICE: {
```

Comment:
The user might want to customize this because this is a spam protection mechanism only : if the user wants to use their own API node with a lower COSMOS_GAS_PRICE, we should allow them to modify it here so that the automatic fee estimation can directly use this value to make the computation

Reply:
ok for now, but for long term we need to consider the concept of per family/account/currency settings that we expose to UI.